support/SubmitScripts/Minerva.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -
+#SBATCH -J spectre
+#SBATCH --nodes 2
+#SBATCH -t 24:0:00
```

Comment:
Can you confirm (as a response here) that this works and/or fix it?

Reply:
Oh, apparently it does, but I fixed it